wp-content/themes/core/components/header/subheader/Subheader_Controller.php

```diff
@@ -50,8 +71,71 @@ public function get_title_args(): array {
 
 		return [
 			Text_Controller::TAG     => 'h1',
-			Text_Controller::CLASSES => [ 'page-title', 'h1' ],
+			Text_Controller::CLASSES => [ 'page-title', 'h1', 'c-subheader__title' ],
 			Text_Controller::CONTENT => $this->get_page_title(),
 		];
 	}
+
+		/**
+	 * @return string
+	 */
+	public function get_container_classes(): string {
+		return Markup_Utils::class_attribute( $this->container_classes );
+	}
+
+	/**
+	 * @return string
+	 */
+	public function get_media_classes(): string {
+		return Markup_Utils::class_attribute( $this->media_classes );
+	}
+
+	/**
+	 * @return string
+	 */
+	public function get_content_classes(): string {
+		return Markup_Utils::class_attribute( $this->content_classes );
+	}
+
+	public function get_image_args() {
+		if ( ! has_post_thumbnail() ) {
+			return [];
+		}
+
+		return [
```

Comment:
Thought here: We rarely use the post thumbnail as the subheader image as strategy/design generally like to have separate images for a post in loops versus in the subheader. Let's bring this up with design and see if they have a preference for using post thumbnail versus adding a subheader image field.

Reply:
@kyleunzicker ^

Reply:
I like this idea ^